library/Octane/Utility/ClassPropertyMap.hs

```
@@ -61,13 +61,19 @@ getClassCache :: Replay.ReplayWithoutFrames
 getClassCache replay = do
   let classNames = replay & getActorMap & Bimap.toMapR
   replay & #cache & #unpack &
-    Maybe.mapMaybe
+    map
       (\cacheItem -> do
          let classId = cacheItem & #classId & Word32.fromWord32
-         className <- Map.lookup classId classNames
+         let className =
+               case Map.lookup classId classNames of
+                 Nothing ->
+                   error
+                     ("could not find class name for id " ++
+                      show classId ++ " in " ++ show classNames)
```

This blows up instead of silently ignoring failures now.